src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/impl/RequestResolver.java

```diff
@@ -349,7 +349,7 @@ private static void addIndicesSourceSafe(final AuditMessage msg,
             boolean resolveIndices,
             final boolean addSource,
             final boolean sourceIsSensitive,
-            final String opendistrosecurityIndex) {
+            final String openSearchSecurityIndex) {
```

Comment:
Sorry, missed this variable. Need to rename this too.

Reply:
Done